libs/ngx-charts-on-fhir/project.json

```diff
@@ -8,8 +8,7 @@
     "build": {
       "executor": "@nx/angular:package",
       "options": {
-        "project": "libs/ngx-charts-on-fhir/ng-package.json",
-        "updateBuildableProjectDepsInPackageJson": true
```

Comment:
`updateBuildableProjectDepsInPackageJson` is deprecated so remove this, alternative for this I have added `@nx/dependency-checks ESLint` and also created `.eslintrc.json` to add the rules